saleor/plugins/invoicing/plugin.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Optional
+from uuid import uuid4
+
+from django.core.files.base import ContentFile
+
+from ...core import JobStatus
+from ...invoice.models import Invoice
+from ...order.models import Order
+from ..base_plugin import BasePlugin
+from .utils import generate_invoice_number, generate_invoice_pdf
+
+
+class InvoicingPlugin(BasePlugin):
+    PLUGIN_ID = "mirumee.invoicing"
+    PLUGIN_NAME = "Invoicing"
+    DEFAULT_ACTIVE = True
```

Comment:
Just to be sure. Did you test the functionality with the scenario when the user has turned off `InvoicingPlugin` and he doesn't have any other Apps/Plugins for invoicing?

Reply:
If this is what the user wants to do, it is an intentional configuration. No invoice will be generated in that case but if this is how system is configured it should work this way I assume :man_shrugging:

Reply:
So my question was. Did you test this scenario? It is a new functionality we need to assume that someone will turn it off.

Reply:
Tested, doesn't break anything other that invoice is not being generated.